src/components/Firestore/Requests/RequestDetails/CodeViewer/index.scss

```
@@ -81,3 +85,140 @@
     width: 61px !important;
   }
 }
+
+/* CodeMirror theme, based on http://go/firebase-console-codemirror-theme */
+
+.cm-s-devsite-light {
+  background-color: white;
+  color: #374747;
+}
+
+.cm-s-devsite-light .CodeMirror-cursor {
+  border-left: 1px solid #374747;
+}
+
+.cm-s-devsite-light div.CodeMirror-selected {
+  background-color: #ebebeb;
+}
+
+.cm-s-devsite-light.CodeMirror-focused div.CodeMirror-selected {
+  background-color: #ebebeb;
+}
+
+.cm-s-devsite-light .CodeMirror-line::selection,
+.cm-s-devsite-light .CodeMirror-line > span::selection,
+.cm-s-devsite-light .CodeMirror-line > span > span::selection {
+  background-color: #ebebeb;
+}
+
+.cm-s-devsite-light .CodeMirror-line::-moz-selection,
+.cm-s-devsite-light .CodeMirror-line > span::-moz-selection,
+.cm-s-devsite-light .CodeMirror-line > span > span::-moz-selection {
+  background-color: #ebebeb;
+}
+
+.cm-s-devsite-light .CodeMirror-activeline-background {
+  background-color: rgba(
+    3,
+    155,
+    229,
+    0.1
+  ); /* GMP_COLOR_PRIMARY_DEFAULT @ 10% */
```

Let's get rid of this comment
```suggestion
  );
```